### ISSUE TITLE
Accept exposant for integer

### ISSUE THREAD

**Issue by FauconFan** [Heijix/nut-lang#26]
Do we accept

10e10

e10

e2

**Comment by xavitator** [comment]
I think that exponent is only after a float number.
Moreover, the number after the exponent have to be a int8 number.
What do you think about it ?

**Comment by FauconFan** [comment]
Json accept e|E after every number. We should accept it...

**Comment by maiste** [comment]
I don't like to put an exponent after an int number but if JSON accepts it, we must follow this logic...

**Comment by xavitator** [comment]
that's right, json accepts this. So we need to accept. In conclusion, we need to put it in the grammar.
